roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=missing-docstring
+
+"""Interface to OpenShift oc command"""
+
+import os
+import shlex
+import shutil
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+ADDITIONAL_PATH_LOOKUPS = ['/usr/local/bin', os.path.expanduser('~/bin')]
+
+
+def locate_oc_binary():
+    ''' Find and return oc binary file '''
```

Comment:
:+1: OK I didn't check the PEP and I got it wrong.

Should we also agree on regular quoted strings? Although it's not specified in a PEP I thought the standard there was single quotes...